python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -1220,6 +1224,10 @@ def execute_asset_backfill_iteration_inner(
                 " AssetGraphSubset"
             )
 
+        logger.info(
```

Comment:
this should make it faster to understand which new materialization events the current tick is considering. previously you'd need to manually compare the materialized subsets for two different ticks.

general logic is that new materializations are what will cause the backfill to submit new runs, so being able to easily see which new materializations are being considered can provide some more information